README.md

```diff
@@ -1,13 +1,11 @@
 <p align="center" id="reactpy">
-    <a href="https://reactpy.herokuapp.com">
```

Comment:
I think clicking the logo ought to take you to the documentation, no?

Reply:
It isn't exactly expected that a logo in a GitHub readme is clickable so it would get infrequent use.

Also, would make it a bit awkward for the other repos. For example, directly linking to the ReactPy core docs from the django-reactpy repo would cause confusion. But also changing the link to the django-reactpy docs within that repo also feels weird.

Reply:
I guess, but images in the readme are clickable by default. It feels weird to see that the logo is clickable and then be taken to the image source. For example, Starlette's logo on their README (which is where I copied the layout from) links the logo to their docs.

I think the issue with linking via the logo in the Django repo is that the logo just says "ReactPy" instead of "ReactPy for Django" or something like that. If it were the latter, it would make sense for it to link to the Django specific docs. With it just being "ReactPy", I agree, that for the Django repo, it's unclear where it should link, if at all.

Reply:
We can disable the github image link behavior by setting `<a href="#">`.

To be honest, I'm undecided on which option to go with though. But it is a lot more convenient if we just stick with a single, unclickable logo across all repos.

Reply:
On this note, should we remove the hyperlink on the word `ReactPy` in the description if it might make the `juptyer`/`django`/`plotly` experience feel weird?

Reply:
I think it needs to be there. The only other link in that paragraph is to ReactJS.

Reply:
Maybe we change that reactpy link to a pypi or GitHub URL instead of docs?

Reply:
What's the issue with linking to docs?

Reply:
I think that's a good observation.